README.md

```diff
@@ -263,6 +263,10 @@ By default Jest ignores everything in `node_modules`. This setting prevents Jest
   "moduleDirectories": ["node_modules", "<path_to_your_sources>"]
 }
 ```
+### TS compiler && error reporting
+- Only syntactic errors are reported by the [tsc `transpileModule`](https://github.com/Microsoft/TypeScript/issues/4864#issuecomment-141567247)
+- Non syntactic errors do not show up in [Jest](https://github.com/facebook/jest/issues/2168)  
+- A workaround is to call the ts compiler before calling Jest `tsc --noEmit -p . && jest`
```

Comment:
I think in a lot of cases you want to run the tests even though tsc reports errors. Maybe last line could be 'If you only want to run jest if tsc does not output any errors, a workaround is ...' ?